client/src/api/api_client.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+/**
+ * Client to interface with API.
+ */
+class ApiClient {
+    // Retrieve geo ids for API.
```

Comment:
nit: Retrieve geo ids of the geographic entities which intersect with the lat, long pair. (or something like this)

Reply:
Done